azure-pipelines.yml

```diff
@@ -469,3 +469,32 @@ jobs:
       addChangeLog: false
       assets: |
           $(Build.ArtifactStagingDirectory)/dist-final/*
+
+
+- job: build_docker_containers
+  displayName: "Build Docker Containers"
+  pool:
+    vmImage: ubuntu-16.04
+  dependsOn:
+    - lint
+  steps:
+  - task: Docker@2
+    displayName: Build Developer Utility Container
+    inputs:
+      command: build
+      tags: opentitan
+      Dockerfile: ./util/container/Dockerfile
+      buildContext: .
+  - task: Docker@2
+    displayName: Build Documentation Builder Container
+    inputs:
+      command: build
+      tags: gcr.io/active-premise-257318/builder
```

Comment:
This doesn't need to be the full name because it's never being pushed.

Reply:
I was thinking about this, and want to make it easier to push later, when we come to it. Maybe this is a silly choice.

Reply:
Even if it was pushed it would typically be tagged twice, once with a "normal" descriptive tag, and once with the GCP-specific tag. So I'd leave the tag out here.